compiler/util/files.cpp

```diff
@@ -424,13 +425,48 @@ const char* createDebuggerFile(const char* debugger, int argc, char* argv[]) {
   return dbgfilename;
 }
 
-const std::string runUtilScript(const char* script) {
+std::string runPrintChplEnv(std::map<std::string, const char*> varMap) {
+  // Run printchplenv script, passing currently known CHPL_vars as well
+
+  std::string command = "";
+  std::string result = "";
+  char buffer[256];
+
+  // Pass known variables into printchplenv via command
+  for( std::map<std::string, const char*>::iterator ii=varMap.begin(); ii!=varMap.end(); ++ii)
+  {
+    command += ii->first + "=" + std::string(ii->second) + " ";
+  }
+
+  command += std::string(CHPL_HOME) + "/util/printchplenv --simple";
+
```

Comment:
Whoops, I completely missed that. I thought runUtilScript was just a helper routine to run any generic utility script, not just one in the util/ dir

Is it worth pulling out the logic for popen/read/pclose into a seperate function?

Actually it looks like runUtilScript was only used for running printchplenv so maybe just delete it?